src/pages/editor/PublishStrategySettings.tsx

```diff
@@ -71,74 +93,207 @@ export const PublishStrategySettings = () => {
+                  <Field name={'rightsRetentionStrategy.type'}>
+                    {({ field }: FieldProps<RightsRetentionStrategyTypes>) => (
+                      <FormControlLabel
+                        label={t('editor.retentions_strategy.rights_retentions_strategy')}
+                        control={
+                          <Checkbox
+                            data-testid={dataTestId.editor.rightsRetentionsStrategy}
+                            {...field}
+                            disabled={isSubmitting}
+                            checked={isRrs || isOverridableRrs}
+                            value={values.rightsRetentionStrategy?.type}
+                            onChange={() => {
+                              setFieldValue(
+                                field.name,
+                                isRrs || isOverridableRrs
+                                  ? RightsRetentionStrategyTypes.NullRightsRetentionStrategy
+                                  : RightsRetentionStrategyTypes.RightsRetentionStrategy
+                              );
+                              setFieldValue('rightsRetentionStrategy.id', '');
+                            }}
+                          />
+                        }
+                      />
+                    )}
+                  </Field>
+
+                  <Field name="rightsRetentionStrategy.id">
+                    {({ field }: FieldProps<string>) => (
+                      <FormLabel
+                        component="legend"
+                        sx={{ color: 'primary.main', fontWeight: 'bold', marginTop: '1rem' }}>
+                        {t('editor.retentions_strategy.institution_rrs_info_page')}
+
+                        <TextField
+                          type="url"
+                          data-testid={dataTestId.editor.rightsRetentionsStrategyLink}
+                          {...field}
+                          label={t('editor.retentions_strategy.institution_rrs_url')}
+                          required={isRrs || isOverridableRrs}
+                          placeholder={t('editor.retentions_strategy.institution_rrs_link')}
+                          variant="filled"
+                          fullWidth
+                          disabled={isNullRrs || isSubmitting}
+                          InputProps={{
+                            startAdornment: (
+                              <InputAdornment position="start">
+                                <LinkIcon />
+                              </InputAdornment>
+                            ),
+                          }}
+                        />
+                      </FormLabel>
+                    )}
+                  </Field>
+
+                  <Typography sx={{ fontStyle: 'italic', marginTop: '0.5rem', marginBottom: '1rem' }}>
+                    {t('editor.retentions_strategy.institution_rrs_required_link')}
+                  </Typography>
```

Comment:
Se om `RequiredDescription`-komponenten kan gjenbrukes her
```suggestion
                  <RequiredDescription />
```